src/libraries.sh

```diff
@@ -487,6 +488,15 @@ rlImport() {
     if ! eval $VERIFIER
     then
       rlLogError "rlImport: Import of library $LIBRARY was not successful (callback failed)"
+      if ! declare -F $VERIFIER > /dev/null
+      then
+        rlLogInfo "rlImport: Callback function $VERIFIER does not exist (see lib.sh: '#   library-prefix = $PREFIX')"
+        local FOO=$(sed -r 's/#.*//' $LIBFILE | grep -Eo '\w+LibraryLoaded\s*\(|function\s+\w+LibraryLoaded\b' | grep -Eo "\w+LibraryLoaded" | sort -u )
+        for F in $FOO
```

Comment:
not to affect test's score we need to use as many local variables as possible.
```suggestion
        local F
        for F in $FOO
```